src/platform/ESP32/PlatformManagerImpl.cpp

```diff
@@ -150,6 +150,8 @@ void PlatformManagerImpl::_Shutdown()
     }
 
     Internal::GenericPlatformManagerImpl_FreeRTOS<PlatformManagerImpl>::_Shutdown();
+
+    esp_event_loop_delete_default();
```

Comment:
Is any kind of additional testing (apart from the Qemu test) required for this?

Reply:
This depends on whether PlatformManager::Shutdown is called outside of the qemu tests...